### PR TITLE
new Overview design screen

### PR DESCRIPTION
**Pull request by almeynman**

### CODE REVIEW

File: src/overview/currency/Currency.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const isValidCurrency = currency => !!currencyIcons[currency];
+
+export const CurrencyIcon = ({ currency }: Props) => {
+  const Icon = currencyIcons[currency];
+  return currencyIcons[currency] ? <Icon /> : null;
+};
+
+const FLexContainer = styled.div`
```
Comment (v3rm0n) [inline src/overview/currency/Currency.js]: Typo

File: src/overview/currency/Currency.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import variables from '../../ui/variables';
+import BitcoinIcon from './icons/BitcoinIcon';
+import EtherIcon from './icons/EtherIcon';
+import LitecoinIcon from './icons/LitecoinIcon';
+
+type Props = {
+  currency: string,
+};
+
+const currencyIcons = {
```
Comment (v3rm0n) [inline src/overview/currency/Currency.js]: Think about how to solve adding new currencies easily

File: src/overview/currency/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import Currency from './Currency';
+
+export { default as Currency } from './Currency';
+
+export default Currency;
```
Comment (v3rm0n) [inline src/overview/currency/index.js]: Try to remove default exports

File: src/overview/currency/Currency.js (hunk 1 above)
Comment (almeynman) [reply src/overview/currency/Currency.js]: fixed

File: src/overview/currency/index.js (hunk 3 above)
Comment (almeynman) [reply src/overview/currency/index.js]: I think we should agree on it during the tech workshop, I have added to my tasks in asana

**Pull request merged**